src/main/scala/dxWDL/dx/DxFindDataObjects.scala

```diff
@@ -144,24 +144,33 @@ case class DxFindDataObjects(limit: Option[Int], verbose: Verbose) {
     JsObject(part1 ++ part2 ++ part3)
   }
 
-  // Submit a request for a limited number of objects
-  private def submitRequest(
-      scope: JsValue,
-      dxProject: DxProject,
+  // Create a request for a limited number of objects
+  private def createRequest(
+      scope: Option[JsValue],
+      dxProject: Option[DxProject],
       cursor: Option[JsValue],
       klass: Option[String],
       propertyConstraints: Vector[String],
       nameConstraints: Vector[String],
-      withInputOutputSpec: Boolean
-  ): (Map[DxDataObject, DxObjectDescribe], Option[JsValue]) = {
+      withInputOutputSpec: Boolean,
+      idConstraints: Vector[String],
+      extraFields: Vector[String]
+  ): JsObject = {
     var fields = Set(Field.Name, Field.Folder, Field.Size, Field.ArchivalState, Field.Properties)
+    fields ++= extraFields.map(field => Field.withName(field)).toSet // TODO: This is case sensitive, is this ok?
```

Comment:
IMO the user will expect they can supply the field name strings with casing as defined by the `system/findDataObjects` method, so this mapping would then fail. Would it make sense to have a Vector or Set of Field values? Like [here](https://github.com/dnanexus/dxWDL/blob/master/src/main/scala/dxWDL/dx/DxFile.scala#L48) for example

Reply:
Fixed.